web/react/components/channel_loader.jsx

```diff
@@ -90,11 +90,12 @@ export default class ChannelLoader extends React.Component {
         }
 
         /* Setup global mouse events */
-        $('body').on('click.userpopover', function popOver(e) {
-            if ($(e.target).attr('data-toggle') !== 'popover' &&
-                $(e.target).parents('.popover.in').length === 0) {
-                $('.user-popover').popover('hide');
-            }
+        $('body').on('click', function hidePopover(e) {
+            $('[data-toggle="popover"]').each(function eachPopover() {
+                if (!$(this).is(e.target) && $(this).has(e.target).length === 0 && $('.popover').has(e.target).length === 0) {
+                    $(this).popover('hide');
+                }
+            });
```

Comment:
We should replace these popovers with react-bootstrap versions. But not in this PR.